File: php/2nd_week/Inheritance/Karyawan.php
<?php
class Karyawan extends Manusia{

    private $nomor_pegawai;
    private $jabatan;
    private $departemen;

    function __construct()
    {
    }

    function setNomorPegawai($nomor){
        $this->nomor_pegawai = $nomor;
    }

    function setJabatan($jabatan){
        $this->jabatan = $jabatan;
    }
    
    function setDepartemen($departemen){
        $this->departemen = $departemen;
    }

    function getNomorPegawai(){
        return $this->nomor_pegawai;
    }

    function getJabatan(){
        return $this->jabatan;
    }
    
    function getDepartemen(){
        return $this->departemen;
    }

    function __destruct()
    {
    }
}

?>

File: cpp/2nd_Week/Inheritance/Karyawan.cpp
#include <iostream>
#include <string>
#include "Manusia.cpp"
using namespace std;

class Karyawan : public Manusia
{
private:
    string no_pegawai;
    string jabatan;
    string departemen;
public:

    Karyawan(){};

    void setNomorPegawai(string nomor){
        this->no_pegawai = nomor;
    }

    void setJabatan(string jabatan){
        this->jabatan = jabatan;
    }
        
    void setDepartemen(string departemen){
        this->departemen = departemen;
    }

    string getNomorPegawai(){
        return this->no_pegawai;
    }

    string getJabatan(){
        return this->jabatan;
    }

    string getDepartemen(){
        return this->departemen;
    }

    ~Karyawan(){};
};